src/main/java/ch/hevs/cloudio/endpoint/CloudioMessageFormatFactory.java

```diff
@@ -2,13 +2,33 @@
 
 class CloudioMessageFormatFactory {
     static CloudioMessageFormat json;
+    static CloudioMessageFormat cbor;
 
     static CloudioMessageFormat massageFormat(int messageFormatId) {
```

Comment:
`messageFormat()`, not `massageFormat()` 🤣 . Probably I went to the physiotherapist that day...